Contributing.md

```diff
@@ -399,6 +399,8 @@ These labels indicate the level or type of experience needed to address. Example
 
 * `W-consensus_breaking` - This PR changes consensus rules and would require a hard-fork to implement.
 * `W-network_breaking` - This PR changes networking messages and requires care when deploying.
+* `W-transaction_breaking` - This PR changes data that wallets use to send transactions. This might not cause 
```

Comment:
Perhaps `W-chainstorage_breaking` - This PR makes incompatible changes to the chainstorage database. This may cause nodes to reject valid blocks.

Reply:
Yeah there are a lot. Also recovery breaking. I don't want to add too many specifics